app/client/components/FilterMenu.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button'
+import Dropdown from 'react-bootstrap/Dropdown'
+import DropDownButton from 'react-bootstrap/DropdownButton'
+import { CaretDownFill, Filter } from 'react-bootstrap-icons'
+import { Link, withRouter } from 'react-router-dom';
+import { withRemote } from '../contexts/RemoteContext.jsx';
+import { withRoutes } from '../contexts/RouteContext.jsx';
+import { withSession } from '../contexts/SessionContext.jsx';
+import selectors from './FilterMenu.module.scss'
+
+/** ListingContainer */
+class FilterMenu extends React.Component {
+  /** @param {Object} props */
+  constructor(props) {
+    super(props);
+  }
+
+  /** @return {JSX.Element} */
+  render() {
+    return (
+      <div id={selectors['filter-container']}>
+        <Button id={selectors['filter-button']}><Filter id={selectors['filter-icon']}/>Filters<CaretDownFill id={selectors['arrow-icon']}/></Button>
+          <div id={selectors['sort-container']}>
+          <div id={selectors['results-container']}>
+          <h6 id={selectors['results-text']}>20,730 results</h6>
+          </div>
+          <div id={selectors['dropdown-container']}>
+          <DropDownButton id={selectors['sort-button']} title="Sort By ">
+            <Dropdown.Item>Relevance</Dropdown.Item>
+            <Dropdown.Item>Newest First</Dropdown.Item>
+            <Dropdown.Item>Oldest First</Dropdown.Item>
+            <Dropdown.Item>Salary Increasing</Dropdown.Item>
+            <Dropdown.Item>Salary Decreasing</Dropdown.Item>
+            <Dropdown.Item>Experience Increasing</Dropdown.Item>
+            <Dropdown.Item>Experience Decreasing</Dropdown.Item>
+          </DropDownButton>
+          </div>
+          </div>
+      </div>
+    );
+  }
+}
+
+export default withRemote(withSession(withRoutes(withRouter(FilterMenu)))

```

Comment:
I'm guessing this is a copy from other files - you actually don't need these here. They're actually composition hooks that make "global" properties available within the component tree (i.e. `withRemote` exposes the remote instance contained in `App` available as `this.props.remote` here). So, if you don't need to use them within the component the composition isn't necessary.

```suggestion
export default FilterMenu
```